.goreleaser.yml

```
@@ -108,6 +108,7 @@ dockers:
       - hack/addons/mindthegap-helm-registry/repos.yaml
     build_flag_templates:
       - "--platform=linux/amd64"
+      - '--build-arg=version=v{{ trimprefix .Version "v" }}'
```

Isnt it case sensetive?
```suggestion
      - '--build-arg=VERSION=v{{ trimprefix .Version "v" }}'
```

---

this seems to compile just fine for me